doc/old_and_new_proposal.md

```diff
@@ -108,18 +178,304 @@ addition, every volume gets a "weight" (so far, based on the
+  * `default_encrypt`. Whether encryption should be used by default.
+  * `default_windows_delete`. Default value for the automatic delete mode for
+    Windows partitions. It can be `none`, `all` or `ondemand`. For more
+    information, see the description of the new proposal above.
+  * `default_linux_delete`. Default value for the automatic delete mode for
+    Linux partitions. See `default_windows_delete` above.
+  * `default_other_delete`. Default value for the automatic delete mode for
+    other partitions. See `default_windows_delete` above.
+  * `default_resize_windows`. Default value for the user setting deciding
+    whether to resize Windows systems if needed.
+  * `lvm_vg_strategy`. If the user decides to use LVM, strategy to decide the
+    size of the volume group (and, thus, the number and size of created physical
+    volumes). It can be `use_available`, `use_needed` or `use_vg_size`. See the
+    description of the new proposal above.
+  * `lvm_vg_size`. If `use_vg_size` is specified in the previous option, this
+    will specify the predefined size of the LVM volume group.
+
+### The `volumes` subsection
+
+Another new `volumes` subsection will be responsible of specifying the
+partitions (or logical volumes if LVM is chosen) to create during the proposal
+and also the behavior of the expert partitioner regarding them.
+
+It will be basically a collection of `volume` subsections, each of them with the
+options listed here. Having read the "How the new proposal distributes the
+space" may be important to fully understand some of them.
+
+  * `mount_point`. Directory where the volume will be mounted in the system.
+  * `fstypes`. A collection of acceptable file system types. If no list if
+    given, YaST will use a fallback one based on the mount point.
+  * `default_fstype`. Default file system type to format the volume.
+  * `desired_size`. Initial size to use in the first proposal attempt.
+  * `min_size`. Initial size to use in the second proposal attempt.
+  * `max_size`. Maximum size to assign to the volume. It can also contain the
+    value "unlimited" (meaning as big as possible). This will be considered the
+    default value if the option is not present.
+  * `max_size_lvm`. When LVM is used, this option can be used to override the
+    value at `max_size`.
+  * `weight`. Value used to distribute the extra space (after assigning the
+    minimal ones) among the volumes.
+  * `adjust_by_ram`. Whether the initial and max sizes of each attempt should
+    be adjusted based in the RAM size. Possible values are "true", "false",
+    "optional", with "false" being the default if not present and "optional"
+    meaning the user can decide via UI. So far the adaptation consist in
+    ensuring all the sizes are, at least, as big as the RAM. In the future, an
+    extra `adjust_by_ram_mode` option could be added to allow other approaches.
+  * `default_adjust_by_ram`. If the previous value is "optional", whether to
```

Comment:
This option looks a little bit weird to me. I would expect something like `adjust_by_ram` with values `true` or `false`, and then an option `optional_adjust_by_ram` with values `true` or `false` meaning that the user can change the value.